treeherder/auth/backends.py

```diff
@@ -187,18 +200,25 @@ def authenticate(self, request):
 
         user_info = self._get_user_info(access_token, id_token)
         username = self._get_username_from_userinfo(user_info)
+        is_sheriff = self._get_is_staff_from_userinfo(user_info)
```

Comment:
this will need to be changed as well s/staff/sheriff/

Reply:
Thanks, didn't amend the latest change.